toc/working-groups/contributions-for-user.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+USERNAME=$1
+shift
+WORKING_GROUP="$*"
+
+echo "Parsing working group metadata..."
+TOC_JSON=$(./parsable-working-groups.sh)
+mapfile -t WORKING_GROUPS <<< "$(echo "${TOC_JSON}" | jq -r '.[].name')"
```

Comment:
By default the macOS users has an older version of the `bash` something like `version 3.2...` which doesn't support `mapfile`. Is there a chance to change the script to work without `mapfile`?

Reply:
I ran into this as well, it's not there on macOS Sonoma.

Reply:
Ah. yeah, I use the bash provided from homebrew. 

i'll see if there are other alternatives that can work cross-platform.

Reply:
I tried using IFS + direct array creation, much to the chagrin of shellcheck. But it looks like it's working.

Reply:
Thanks!